front/src/modules/companies/board/components/CompanyBoard.tsx

```diff
@@ -36,6 +38,8 @@ export const CompanyBoard = ({
 }: CompanyBoardProps) => {
   const viewScopeId = 'company-board-view';
 
+  const { persistViewFields } = useView({ viewScopeId });
```

Comment:
@bosiraphael could we import it from useViewFields. I know that this is not the right way to do it but I'm worried calling useView will cause a re-render on CompanyBoard, we can re-discuss that while refactoring the Board